inference-engine/src/readers/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+set(TARGET_NAME inference_engine_reader_api)
+
+# Reader API library
+add_library(${TARGET_NAME} INTERFACE)
+
+target_include_directories(${TARGET_NAME} INTERFACE
+    "${CMAKE_CURRENT_SOURCE_DIR}/reader_api"
+    ${PUBLIC_HEADERS_DIR})
```

Comment:
is `PUBLIC_HEADERS_DIR` set?

Reply:
Probably, we need to use `$<TARGET_PROPERTY:inference_engine,INTERFACE_INCLUDE_DIRECTORIES>`

Reply:
Fixed.